frontend/sw360-portlet/src/main/webapp/css/external/jquery.treetable.css

```diff
@@ -0,0 +1,28 @@
+table.treetable span.indenter {
```

Comment:
These two new css files come from jquery treetable plugin. Do you think I should copy the header from jquery.treetable.js?